Answers.md

```diff
@@ -0,0 +1,9 @@
+# Answers
+
+1. What is React JS? How does it differ from other JavaScript Frameworks?

```

Comment:
This all looks great.  Let's make the F in Frameworks lowercase though and then we can merge.